test/unit_test/api/apps/services/test_dataset_api_service_nav.py

```
@@ -0,0 +1,213 @@
+    for key, value in attrs.items():
+        setattr(mod, key, value)
+    monkeypatch.setitem(sys.modules, name, mod)
+    if "." in name:
+        parent_name, _, child_name = name.rpartition(".")
+        parent_mod = sys.modules.get(parent_name)
+        if parent_mod is not None:
+            monkeypatch.setattr(parent_mod, child_name, mod, raising=False)
+    return mod
+
+
+def _load_nav_module(monkeypatch, *, accessible=True, index_pack=("idx-1", object()), total=0, field_map=None):
```

Ruff (0.16.0)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)